src/main/java/util/IdGeneratorUtil.java

```diff
@@ -4,6 +4,7 @@
 
     private static Long productId = 1L;
     private static Long userId = 1L;
+    private static Long orderId = 100000L;
```

Comment:
why 100000L?

Reply:
Because it's order id (i think it should be a big number
Or no?

Reply:
id need start from 1 to ...

Reply:
Ok, i changed it